legal-api/report-templates/template-parts/common/statement.html

```diff
@@ -0,0 +1,26 @@
+{% if business.legalType == 'BEN' %}
+  <div class="no-page-break">
+    <div class="section-title mt-4">B.C. Benefit Company Statement</div>
```

Comment:
Are we going with "BC" or "B.C." ? I think it's inconsistent atm.

Reply:
Yeah Linda asked me to make this change yesterday.